src/serve_rendered.js

```diff
@@ -503,14 +503,23 @@ const respondImage = (
         image.composite(composites);
       }
 
-      const formatQuality = (options.formatQuality || {})[format];
+      const formatOptions = (options.formatOptions || {})[format] || {};
```

Comment:
Feels like `options.formatQuality` should still be supported (IOW, when people would update to a new version, they suddenly wouldn't get lower or higher qualities than what they expect), maybe log a deprecation warning though?

Reply:
ok, this is done. `formatQuality` is still supported but will produce a deprecation warning if it's found. `formatOptions` is preferred over `formatQuality`.